### PR TITLE
TC API Fixes

### PR DESCRIPTION
**Pull request by Camwyn**
Tweak modal loading behavior. 
Ensure that all providers do their own checks on AR page.
Update common/postcss/utils.
FIx incorrect non-0meta count on AR page.
Formatting and fix for EDD "non-cart".
Fix warning in `event-tickets/src/Tribe/Commerce/PayPal/Main.php` `maybe_delete_expired_products()`

Required by https://github.com/moderntribe/event-tickets-plus/pull/880

### CODE REVIEW

File: src/Tribe/Commerce/PayPal/Main.php [hunk 1]
```diff
@@ -3064,4 +3075,29 @@ public function tribe_attendee_registration_form_class( $classes ) {
...
+	 */
+	function tribe_attendee_registration_cart_provider( $provider_obj, $provider ) {
+		$options = [
+			'tpp',
+			'tribe_tpp_attendees',
+			'tribe-commerce',
+			__CLASS__,
+		];
+
+		if ( in_array( $provider, $options) ) {
```
Comment (sc0ttkclark) [inline src/Tribe/Commerce/PayPal/Main.php]:
```suggestion
		if ( in_array( $provider, $options, true ) ) {
```

File: src/views/modal/registration-js.php [hunk 2]
```diff
@@ -102,15 +102,15 @@ class="tribe-tickets__item__attendee__fields__form <?php echo sanitize_html_clas
...
+					class="tribe-common-c-btn-link tribe-common-c-btn--small tribe-block__tickets__item__attendee__fields__footer_submit tribe-tickets__attendee__fields__footer_cart-button tribe-validation-submit"
 					name="cart-button"
 					>
 						<?php esc_html_e( 'Save and View Cart', 'event-tickets' ); ?>
 				</button>
 				<span class="tribe-block__tickets__item__attendee__fields__footer__divider"><?php esc_html_e( 'or', 'event-tickets' ); ?></span>
 				<button
 					type="submit"
-					class="tribe-common-c-btn tribe-common-c-btn--small tribe-block__tickets__item__attendee__fields__footer_submit tribe-validation-submit"
+					class="tribe-common-c-btn tribe-common-c-btn--small tribe-block__tickets__item__attendee__fields__footer_submit tribe-tickets__attendee__fields__footer_cehckout-button tribe-validation-submit"
```
Comment (Camwyn) [inline src/views/modal/registration-js.php]:
```suggestion
					class="tribe-common-c-btn tribe-common-c-btn--small tribe-block__tickets__item__attendee__fields__footer_submit tribe-tickets__attendee__fields__footer_checkout-button tribe-validation-submit"
```

File: src/resources/js/tickets-registration-page.js [hunk 3]
```diff
@@ -58,7 +58,7 @@ tribe.tickets.registration = {};
 
 	// Bail if there are no tickets on the current event/page/post
 	if ( ! $( obj.selector.eventContainer ).length ) {
-		return;
+		//return;
```
Comment (Camwyn) [inline src/resources/js/tickets-registration-page.js]:
```suggestion
		return;
```

File: src/Tribe/Commerce/PayPal/Main.php [hunk 4]
```diff
@@ -2260,19 +2262,28 @@ public function maybe_delete_expired_products() {
...
+			return;
+		}
 
 		// Bail if ET+ is not in place.
 		if ( ! class_exists( 'Tribe__Tickets_Plus__Meta__Storage' ) ) {
 			return;
 		}
-
 		$storage = new Tribe__Tickets_Plus__Meta__Storage();
-		$storage->delete_cookie();
```
Comment (Camwyn) [inline src/Tribe/Commerce/PayPal/Main.php]: This will fail every time we get here - `delete_cookie()` requires a passed ticket id.

**Pull request merged**